plugins/catalog-backend/src/service/CatalogPlugin.ts

```diff
@@ -99,6 +108,12 @@ export const catalogPlugin = createBackendPlugin({
         });
         builder.addProcessor(...processingExtensions.processors);
         builder.addEntityProvider(...processingExtensions.entityProviders);
+
+        // THIS IS JUST TO ENABLE API DOCS WITH THE NEW BACKEND SYSTEM. DO NOT MERGE THIS.
+        builder.setPlaceholderResolver(
```

Comment:
Before this merges, extension points have to be added to `CatalogPlugin` for support of this resolution.